apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -93,7 +147,7 @@ public static BigtableClient Create(ServiceEndpoint endpoint = null, BigtableSer
         public static BigtableClient Create(Channel channel, BigtableServiceApiSettings settings = null, string appProfileId = null)
         {
             var client = BigtableServiceApiClient.Create(channel, settings);
-            return new BigtableClientImpl(client, appProfileId) { UnderlyingClientSettings = settings };
+            return new BigtableClientImpl(appProfileId, client) { UnderlyingClientSettings = settings };
         }
```

Comment:
Pubsub has a synchronous `Create` which takes an `IEnumerable<T>` of already created clients. We should probably have something similar to be consistent. Although now that I think of it, then we wouldn't be able to set `UnderlyingClientSettings`, because they could all be different. But I think that might actually be ok. Then the usages of it would pick up the default settings.